### DISCUSSION TITLE
mismatch in tag vs artifact naming

### DISCUSSION THREAD

**Discussion by erickellerek1** [git-for-windows/git#4871] [Q&A]
- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

I am doing some automation using your release tag to download and install your release artifacts.
Yet I witnessed that there is some mismatch in the tag naming and the resulting artifact names.

as an example, `v2.44.0.windows.1` will produce an asset name called: `Git-2.44.0-64-bit.exe` with a full link to download: https://github.com/git-for-windows/git/releases/download/v2.44.0.windows.1/Git-2.44.0-64-bit.exe

whereas the tag version: `v2.42.0.windows.2` will produce an asset name called: `Git-2.42.0.2-64-bit.exe` with a full link to download: https://github.com/git-for-windows/git/releases/download/v2.42.0.windows.2/Git-2.42.0.2-64-bit.exe

Sometimes the build increment is embedded in the asset for version > 1
I seems like an odd rule and make the automation unnecessary complex while having to test the build version to ensure the formation of the proper download link.

Any rational about that?

**Comment by dscho** [comment]
If you want to automate anything related to the artifacts, you will want to use the URLs like https://gitforwindows.org/latest-64-bit-installer.url that [are updated as part of Git for Windows' release process](https://github.com/git-for-windows/git-for-windows.github.io/blob/db9a134ed0fd484568124c1f79cb011eddb9a21d/bump-version.js#L23-L30).
